libr/core/cmd_print.c

```diff
@@ -2348,7 +2348,7 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 	bool use_color = r_config_get_i (core->config, "scr.color");
 	int len = 0;
 	int i;
-	int l;
+	int l = 0;
```

Comment:
I think the initialization should be done at the beginning of the body of the for loop.

Reply:
`int l = 0;` at the beginning of the body of the for loop